assets/js/components/Root/index.js

```diff
@@ -50,19 +51,21 @@ export default function Root( {
 			<FeaturesProvider value={ enabledFeatures }>
 				<ErrorHandler viewContext={ viewContext }>
 					<RestoreSnapshots>
-						{ children }
-						{ /*
-							TODO: Replace `FeatureToursDesktop` with `FeatureTours`
-							once tour conflicts in smaller viewports are resolved.
-							@see https://github.com/google/site-kit-wp/issues/3003
-						*/ }
-						{ viewContext && <FeatureToursDesktop viewContext={ viewContext } /> }
-						{ dataAPIContext && (
-						// Legacy dataAPI support.
-							<CollectModuleData context={ dataAPIContext } args={ dataAPIModuleArgs } />
-						) }
+						<HashRouter>
```

Comment:
This seems to be adding a `#` to all URLs where Site Kit apps are rendered

![image](https://user-images.githubusercontent.com/1621608/121661901-5f2efd00-caad-11eb-8270-6f29d7385292.png)

![image](https://user-images.githubusercontent.com/1621608/121661968-75d55400-caad-11eb-890d-000dd3b258db.png)

Can we avoid this except for where it's actively used? If not, we probably need to move the `HashRouter` into the `SettingsApp` for now.

Reply:
Thanks for spotting that! It didn't happen in Storybook strangely, so I fixed it here AND tweaked Storybook so it should fail if that were to happen in the future.

Reply:
Yeah, I moved it for now because realistically we should be keeping it scoped to there for now 👍🏻